clang/lib/CIR/CodeGen/CIRGenDecl.cpp

```diff
@@ -887,6 +887,17 @@ struct CallCleanupFunction final : EHScopeStack::Cleanup {
 };
 } // end anonymous namespace
 
+/// pushDestroy - Push the standard destructor for the given type as
```

Comment:
You can remove "pushDestroy - " from the comment, this is old doxygen stuff we've been already trimming of from Clang/LLVM as code evolves, please remove those when copying function.